addons/website_sale/views/templates.xml

```diff
@@ -1105,18 +1105,25 @@
                                                 Add to cart
                                             </a>
                                         </div>
+                                        <div
```

Comment:
Should be in master I guess, why in 17.0?
If we want to fix in 17 then we can add margin or something like that